text/0001-cdk-update.md

```diff
@@ -0,0 +1,293 @@
+---
+rfc pr: [#335](https://github.com/aws/aws-cdk-rfcs/pull/335)
+tracking issue: https://github.com/aws/aws-cdk-rfcs/issues/1
+---
+
+# CDK Update
+
+Rapid and automatic updates for code-only changes to CDK application stacks.
+
+## Working Backwards
+
+### CHANGELOG
+
+- feat(cli): implement `cdk update` and `cdk update --watch` for asset updates
+
+### README
+
+The `cdk update` command accelerates the asset deployments for your CDK
```

Comment:
As mentioned above - this is not just about assets - we need to make sure we design this to allow adding "shortcut deployments" for any type of resource. Lambda handlers backed by assets are just one example.